katello/katello-restore

```diff
@@ -1,21 +1,32 @@
 #!/usr/bin/env ruby
 
+require 'socket'
 require 'optparse'
 require 'fileutils'
 require 'date'
 
-STANDARD_BACKUP_FILES = ['config_files.tar.gz', 'pulp_data.tar']
+STANDARD_BACKUP_FILES = ['config_files.tar.gz', 'pulp_data.tar*']
 ONLINE_BACKUP_FILES = ['mongo_dump', 'candlepin.dump', 'foreman.dump', 'pg_globals.dump']
 OFFLINE_BACKUP_FILES = ['mongo_data.tar.gz', 'pgsql_data.tar.gz']
+FOREMAN_PROXY_CONTENT_BACKUP_FILES = ['mongo_data.tar.gz']
+FOREMAN_PROXY_CONTENT_ONLINE_BACKUP_FILES = ['mongo_dump']
+FOREMAN_PROXY_CONTENT = "foreman-proxy-content" # keep this variable easy to change for backporting
 
 confirmed = false
+@disable_system_checks = false
+@is_foremnan_proxy_content = !`rpm -qa | grep foreman-proxy-content`.empty?
+@skip_register = false
 
 optparse = OptionParser.new do |opts|
```

Comment:
My apologies. This will need to be changed to an instance variable. I didn't realize the scope gate also involved method definitions.